src/config/config.ts

```diff
@@ -12,7 +12,19 @@ export interface StoreConfig {
     currency: StoreCurrency;
     displayDateFormat: string;
     inputDateFormat: string;
-    formFields: FormFields;
+
+    /**
+     * @deprecated Please use instead the data selectors
+     * @remarks
+     * ```js
+     * const data = CheckoutService.getState().data;
+     * const shippingAddressFields = data.getShippingAddressFields('US');
+     * const billingAddressFields = data.getBillingAddressFields('US');
+     * const customerAccountFields = data.getCustomerAccountFields();
+     * ```
+     */
+    formFields?: FormFields;
```

Comment:
It should still `formFields` (without `?`)

Reply:
fixed